packages/fast-components/src/slider-label/slider-label.template.ts

```diff
@@ -0,0 +1,27 @@
+import { html, when } from "@microsoft/fast-element";
+import { SliderLabel } from "./slider-label";
+import { SliderOrientation } from "../slider";
+
+export const SliderLabelTemplate = html<SliderLabel>`
+    <template
+        class=${x =>
```

Comment:
I think *this* is what Rob was referring to - we don't want to reflect to the "class" attribute here because it will clobber any class attribute applied by an app author.

Reply:
Yes. Hope to fix that but for now, don't set class on the template itself otherwise it will clobber any user classes that they place on the element.

Reply:
dang it I had it in code and moved it here because of another comment. Will put it back